index.html

```diff
@@ -28,7 +28,7 @@
                     <li class="nav-item">
                         <a href="#">Contact</a>
                     </li>
-                    <li class="nav-item">
+                    <li id="hidemail" class="nav-item">
                         <a href="#"><img src="./assets/Disabled.png" alt="mail"></a>
                     </li>
```

Comment:
- [ ] links work well great job. please use the same Figma font weight and size, also the line breaks under the link
- your version
![Screenshot (121)](https://user-images.githubusercontent.com/65298048/173632051-b72e501a-9ee8-43d0-b923-5938ff29a792.png)
- Figma version
![Screenshot (116)](https://user-images.githubusercontent.com/65298048/173632205-045f2f1e-6978-4778-bc73-f7e2dfc9b938.png)

Reply:
Updated